portfolio/src/main/java/com/google/sps/servlets/BlogCommentServlet.java

```diff
@@ -55,8 +57,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       long postId = (Long) entity.getProperty("postid");
       long commentId = (Long) entity.getKey().getId();
       String emoji = (String) entity.getProperty("emoji");
+      String email = (String) entity.getProperty("email");
 
-      Comment comment = new Comment(content, time, name, postId, commentId, emoji);
+      Comment comment = new Comment(content, time, name, postId, commentId, emoji, email, "");
```

Comment:
(entirely optional) If you run out of things to do, explore the Builder pattern; here is a good blog post -- https://medium.com/@ajinkyabadve/builder-design-patterns-in-java-1ffb12648850